electrum/address_synchronizer.py

```diff
@@ -181,9 +182,9 @@ def start_network(self, network: Optional['Network']) -> None:
             self.synchronizer = Synchronizer(self)
             self.verifier = SPV(self.network, self)
             self.asyncio_loop = network.asyncio_loop
-            util.register_callback(self.on_blockchain_updated, ['blockchain_updated'])
+            self.register_callbacks()
 
-    def on_blockchain_updated(self, event, *args):
+    def on_event_blockchain_updated(self, event, *args):
```

Comment:
This would probably raise as the callback has no arguments no, and this expects at least "event".
```suggestion
    def on_event_blockchain_updated(self, *args):
```